tools/webpack-jumpstart

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Download pre-built webpack for current git SHA from GitHub
```

Comment:
I think this should say dist/. As you noticed today, there's dist/guide which isn't a webpack, and the corresponding workflow is also called build-dist, not build-webpack. I wouldn't mind to rename the script accordingly as well (dist-jumpstart, or download-dist), but I feel for this PR I've already used up my quota of name bikeshedding; I can certainly live with the current one :grin: